client/src/services/fake/FakeGameStorage.test.ts

```diff
@@ -0,0 +1,58 @@
+import { FakeGameStorage } from "./FakeGameStorage";
+import Game from "domain/Game";
+
+describe("Fake game storage getGame", () => {
```

Comment:
jei vadinam Fake tai dažniausiai ten quick and dirty parašytas kodas būna, kad pasitestuot kaip integruojasi moduliai, tai jų netestuojam tiesiogiai. Jeigu rašai kažkokią normalią storage implementaciją, tai vadinam su InMemory prefiksu